docs/guides/run-jobs-session.mdx

```diff
@@ -72,14 +70,35 @@ with Session(backend=backend, max_time="25m"):
   ...
 ```
 
-There is also an interactive time to live (ITTL) value that cannot be configured.  If no session jobs are queued within that window, the session is temporarily deactivated.
+There is also an interactive time to live (interactive TTL) value that cannot be configured.  If no session jobs are queued within that window, the session is temporarily deactivated.
+
+Default values:
+
+| Instance type (Open or Premium Plan) |  Interactive TTL | Maximum TTL |
+| --- | --- | --- |
+| Premium Plan | 60 sec* | 8 h* |
+| Open Plan | sessions run in job mode | sessions run in job mode |
 
-To determine a session's max TTL or ITTL, follow the instructions in [Determine session details](#session-details) and look for the `max_time`or `interactive_timeout` value, respectively.
+*\* Certain Premium Plan instances might be configured to have a different value.*
 
-For more information about these settings, see [Session maximum execution time.](/guides/max-execution-time#session-max-time)
+To determine a session's max TTL or  interactive TTL, follow the instructions in [Determine session details](#session-details) and look for the `max_time`or `interactive_timeout` value, respectively.
+
+<span id="ends"></span>
+## End a session
+
+A session ends in the following circumstances:
+
+* The maximum timeout (TTL) value is reached, resulting in the cancellation of all queued jobs.
+* The session is manually canceled, resulting in the cancellation of all queued jobs.
+* The session is manually closed.  The session stops accepting new jobs but continues to run queued jobs with priority.
+* If you use Session as a context manager, that is,  `with Session()`, the session is automatically closed when the context ends (the same behavior as using `session.close()`).
+
+<Admonition type = "note">
+When a session ends, the QPU finishes running any job that is currently running.
+</Admonition>
```

Comment:
Should we remove this? Since the behavior is different between IQP and Paygo, and it's already mentioned in the intro.